ui/packages/consul-ui/app/components/topology-metrics/index.js

```diff
@@ -104,5 +117,17 @@ export default class TopologyMetrics extends Component {
 
     // Set Upstream Cards Positioning points
     this.upLines = this.drawUpLines(upCards);
+
+    // Calculate viewBox dimensions
+    this.downView = document.getElementById('downstream-lines').getBoundingClientRect();
+
+    const upstreamLines = document.getElementById('upstream-lines').getBoundingClientRect();
+    const upstreamColumn = document.getElementById('upstream-column').getBoundingClientRect();
+    this.upView = {
```

Comment:
I had to make some height adjustments. I'll probably have to rework on this again for the GA dropdown.